tool/speed.cc

```diff
@@ -2302,6 +2309,49 @@ static bool parseCommaArgumentToGlobalVector(std::vector<size_t> &vector,
     return true;
 }
 
+static bool parseCommaArgumentToGlobalVector(std::vector<std::string> &vector,
```

Comment:
Is there a nice way to avoid this duplication between the two `parseCommaArgumentToGlobalVector` `size_t` and `string`, and the two `printInputCommaSeparated` with `string` and `size_t`?

Reply:
Split up processing in two pieces:
1. parse comma-separated strings
2. parse to integers if needed